recipes/poppler/all/conanfile.py

```diff
@@ -132,125 +126,119 @@ def _minimum_compilers_version(self):
         }
 
     def validate(self):
-        if self.options.fontconfiguration == "win32" and self.settings.os != "Windows":
+        if self.info.options.fontconfiguration == "win32" and self.settings.os != "Windows":
```

Comment:
This pattern was changed recently with Conan 1.54 and the latest beta

the new advice is self.setting in validate and self.info for package_id :)

```suggestion
        if self.options.fontconfiguration == "win32" and self.settings.os != "Windows":
```